integration/monai/README.md

```diff
@@ -0,0 +1,38 @@
+# MONAI Integration
+
+## Objective
+Integration with [MONAI](https://monai.io/)'s federated learning capabilities.
+
+Add `Executor` class to allow using [MONAI bundle](https://docs.monai.io/en/latest/bundle.html) configuration files with MONAI's `ClientAlgo` class.
```

Comment:
Should this be "Add `ClientAlgoExecutor` class"?

Reply:
Thanks. changed.